cms/articles/tests/test_models.py

```diff
@@ -233,26 +234,203 @@ def test_cite_this_page_is_not_shown_when_unticked(self):
         response = self.client.get(self.basic_page_url)
         self.assertNotContains(response, expected)
 
-    def test_breadcrumb_doesnt_containt_series_url(self):
+    def test_breadcrumb_is_shown(self):
+        # Statistical Articles are a special case regarding breadcrumbs the url includes a series page which is not
+        # accessible externally
         response = self.client.get(self.basic_page_url)
-        # confirm that current breadcrumb is there
-        article_series = self.basic_page.get_parent()
-        self.assertNotContains(
-            response,
-            f'<a class="ons-breadcrumbs__link" href="{article_series.url}">{article_series.title}</a>',
-            html=True,
+        mock_url = response.request["PATH_INFO"]
+        mock_url = mock_url.split("/")
+        mock_url = list(filter("".__ne__, mock_url))
+        mock_breadcrumb = self.get_breadcrumbs(response)
+        expected = 'class="ons-breadcrumbs__link"'
+        self.assertContains(response, expected)
+        self.assertEqual(len(mock_url) - len(mock_breadcrumb), 2)
+        self.assertEqual(mock_url[0], mock_breadcrumb[0])
+        self.assertEqual(mock_url[1], mock_breadcrumb[1])
+
+    def get_breadcrumbs(self, response):
+        mock_response = response.content.decode("utf-8").split("\n")
+        mock_hrf = ""
+        for ind, line in enumerate(mock_response):
+            if "breadcrumb" in line:
+                mock_hrf = mock_response[ind + 1].split("/", 1)
+        mock_breadcrumb = mock_hrf[1][:-1]
+        mock_breadcrumb = mock_breadcrumb.split("/")
+        mock_breadcrumb = list(filter("".__ne__, mock_breadcrumb))
+        return mock_breadcrumb
+
+    def test_pagination_is_not_shown(self):
+        response = self.client.get(self.basic_page_url)
+        expected = 'class="ons-pagination__link"'
+        self.assertNotContains(response, expected)
+
+
+class PreviousReleasesWithoutPaginationTestCase(TestCase):
+    # PREVIOUS_RELEASES_PER_PAGE is default value 10
+    total_batch = 9
+
+    @classmethod
+    def setUpTestData(cls, cls_total_batch: int = total_batch):
+        cls.article_series = ArticleSeriesPageFactory(title="Article Series")
+        cls.articles = StatisticalArticlePageFactory.create_batch(cls_total_batch, parent=cls.article_series)
+        cls.previous_releases_url = cls.article_series.url + cls.article_series.reverse_subpage("previous_releases")
+
+    def test_page_content(self):
```

Comment:
suggestion: test that the article URLs and their titles are there.

The markup will change when FE ticket is done, so this will fail immediately.

Reply:
discussed. will address when the FE is updated